### DISCUSSION TITLE
Discharge functionality for open source version

### DISCUSSION THREAD

**Discussion by milke** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#853] [Q&A]
Not really an issue, maybe it can be treated as a feature request… Since discharge functionality is available in downloadable AlDente Free version, I wonder why it isn't available in the open source version as well. Don't get me wrong, I'm the owner of a lifetime license for AlDente Pro, but I was just wondering about functionality difference between free and open source versions.

Keep up the great work 👍🏻

**Comment by MatthiasKerbl** [comment]
Hi @milke,

AlDente Classic (the open-source version) is the first version of AlDente we developed for ourselves and shared online. After more and more people approached us with feature and support requests, we started developing AlDente Pro from scratch and charged some money for it. However, since we wanted to still offer the basic functionality for free we also launched AlDente Free which uses the same code base as AlDente Pro just with limited features. We stopped working on AlDente Classic a while ago and are only focusing on AlDente Pro and AlDente Free.

Hope that clarifies some things.

Since this is more a discussion than an issue, I will convert it into the discussions section.

Cheers,
Matthias
